dags/data_quality_checks_dag.py

```diff
@@ -69,11 +68,14 @@ def slack_failure_notification(context):
     return failed_alert.execute(context=context)
 
 
-@task
+# Always execute this task, even if previous uploads have failed. There can be already existing
+# files in S3 that haven't been processed yet.
+@task(trigger_rule="all_done")
 def get_files_from_s3(bucket, prefix_value):
     s3_hook = S3Hook()
     paths = s3_hook.list_keys(bucket_name=bucket, prefix=prefix_value)
-    return paths
+    # list_keys also returns directories, we are only interested in files for further processing
+    return list(filter(lambda element: element.endswith(".csv"), paths))
```

Comment:
Good idea to filter files ending with `.csv`, in case there are other types of files as well.